templates/run_multidomain_pytest.vars.yml

```diff
@@ -0,0 +1,10 @@
+---
+repofile_url: {{ repofile_url }}
+update_packages: {{ update_packages }}
+trusted_domain: {{ trusted_domain | default(false) }}
+selinux_enforcing: {{ selinux_enforcing }}
+caless: {{ caless | default(false) }}
+fips: {{ fips | default(false) }}
+enable_testing_repo: {{ enable_testing_repo | default(false) }}
+ansible_python_interpreter: /usr/bin/python3
+{% if copr %}copr: "{{ copr }}"{% endif %}
```

Comment:
Instead of defining a new template, you can add this single line to templates/run_pytest.vars.yml:
```
trusted_domain: {{ trusted_domain | default(false) }}
```